Dockerfile.amd64

```diff
@@ -1,4 +1,31 @@
-FROM alpine:3.7
+# Copyright (c) 2018-2019 Tigera, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+ARG GIT_VERSION=unknown
+
+LABEL name="Calico pod2daemon" \
+      vendor="Project Calico" \
+      version=$GIT_VERSION \
+      release="1" \
+      summary="Secure connections from Kubernetes pods to local daemons using FlexVolume + Unix Domain Socket" \
```

Comment:
I'd instead maybe say `FlexVolume installer to setup secure connections from Kubernetes pods to local daemons`.
IDK, maybe

Reply:
That is better, thanks! 19cd0ab